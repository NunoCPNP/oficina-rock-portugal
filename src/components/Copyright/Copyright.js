import useTranslation from 'next-translate/useTranslation'

import { Container } from './Copyright.styles'

const Copyright = () => {
  const { t } = useTranslation('common')

  return (
    <Container data-testid="copyright">
      <div>{t(`copyright`)}</div>
    </Container>
  )
}

export default Copyright
