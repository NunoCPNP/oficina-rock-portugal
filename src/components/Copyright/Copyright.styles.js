import styled from '@emotion/styled'

export const Container = styled.div`
  display: flex;
  justify-content: center;
  padding-bottom: 2rem;
`
