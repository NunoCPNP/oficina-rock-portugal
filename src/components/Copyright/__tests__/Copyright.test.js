import { render } from '@testing-library/react'

import Copyright from '../Copyright'

describe('<Copyright /> spec', () => {
  it('Should match spashot', () => {
    const { container } = render(<Copyright />)

    expect(container).toMatchSnapshot()
  })

  it('Should render translations keys', () => {
    const { getByText } = render(<Copyright />)

    expect(getByText('common:copyright')).toBeInTheDocument()
  })
})
