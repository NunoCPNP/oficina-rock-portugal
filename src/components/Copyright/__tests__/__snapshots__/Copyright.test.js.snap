// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Copyright /> spec Should match spashot 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  padding: 2rem;
}

<div>
  <div
    class="emotion-0"
    data-testid="copyright"
  >
    <div>
      common:copyright
    </div>
  </div>
</div>
`;
